proto/shaman.proto

```diff
@@ -64,25 +64,25 @@ enum ShamanRune {
 	RuneHealingRain 						= 415236;
 	RuneChestOverload          	= 408438;
 	RuneChestShieldMastery 			= 408524;
-	RuneTwoHandedMastery 				=	436367;
+	RuneTwoHandedMastery 				=	436364;
```

Comment:
Checked all of the rune IDs but only the P2 ones were wrong (because I sourced the "learn" spells not the actual spells)